Header header

float32 u1
float32 v1
float32 u2
float32 v2
float32 u3
float32 v3
float32 u4
float32 v4

float32 u1d
float32 v1d
float32 u2d
float32 v2d
float32 u3d
float32 v3d
float32 u4d
float32 v4d

float32 Eu1
float32 Ev1
float32 Eu2
float32 Ev2
float32 Eu3
float32 Ev3
float32 Eu4
float32 Ev4


float32 J
float32 optNUM
float32 Z
float32 Tx
float32 Ty
float32 Tz
float32 Oz
float32 Tcx
float32 Tcy
float32 Tcz
float32 Ocz

float32 dtloop
float32 time
float32 trig
float32 m
float32 xi
float32 ekm2

# float32[] transformed_features

int32 first_min_index
int32 second_min_index

float32 s_bar_x
float32 s_bar_y
float32 s_bar_x_des
float32 s_bar_y_des

float32 transformed_sigma
float32 transformed_sigma_square
float32 transformed_sigma_square_log
float32 sigma_des
float32 sigma_square_des
float32 sigma_log_des

float32 angle_deg_des
float32 angle_des_tan
float32 transformed_tangent
float32 transformed_angle_radian
float32 transformed_angle_deg

float32 traj_s_1
float32 traj_s_2
float32 traj_s_3
float32 traj_s_4

float32 s_des_1
float32 s_des_2
float32 s_des_3
float32 s_des_4

